lib/active_admin/router.rb

```diff
@@ -107,7 +107,7 @@ def define_belongs_to_routes(config)
     end
 
     def define_namespace(config)
-      router.namespace config.namespace.name, config.namespace.route_options.dup do
+      router.namespace config.namespace.name, **config.namespace.route_options.dup do
```

Comment:
production change.

`dup` may still be needed to avoid nested objects in route options